src/test-run/commands/index.js

```diff
@@ -188,6 +188,23 @@ export class TypeTextCommand extends Assignable {
     }
 }
 
+export class NavigateToCommand extends Assignable {
+    constructor (obj) {
+        super(obj);
+
+        this.type = TYPE.navigateTo;
+        this.url  = null;
+
+        this._assignFrom(obj, true);
+    }
+
+    _getAssignableProperties () {
+        return [
+            { name: 'url', type: nonEmptyStringArgument, required: true }
```

Comment:
I think we need to validate protocol as well (it should be `http`, `https`, relative protocol - `//` or relative URL). BTW, I guess we should allow relative urls, I'm not sure if current `navigateTo` implementation in hammerhead supports relative URLs - we need to check this and create a separate issue for this one.